server/config.go

```diff
@@ -321,6 +323,12 @@ func s3Flags() []cli.Flag {
 			Value:   "",
 			EnvVars: prefixEnvVars("S3_ENDPOINT"),
 		},
+		&cli.BoolFlag{
+			Name:    S3DisableTLSFlagName,
+			Usage:   "whether to disable TLS for S3 storage",
+			Value:   true,
+			EnvVars: prefixEnvVars("S3_DISABLE_TLS"),
+		},
```

Comment:
I personally don't like having a BoolFlag set to true by default, because then actually setting the flag `--s3.disable-tls` is useless, so the only flag config one would ever use is `--s3.disable-tls=false` which feels counterintuitive.

I'd personally prefer to flip to `--s3.enable-tls`, but @epociask wdyt?

Reply:
I tried to keep a similar syntax to `--eigenda-disable-tls` but it's true and logic your point. I'll follow your suggestion, and we can revert or change to something else if @epociask prefers in other way.